components/CalloutBox/index.js

```diff
@@ -0,0 +1,36 @@
+import CalloutBoxButton from "./CalloutBoxButton";
+
+const CalloutBox = ({
+  heading,
+  caption,
+  imgUrl,
+  imgAltText,
+  buttonText,
+  buttonClickHandler,
+}) => {
+  return (
+    <div className="md:h-96 max-w-md mx-auto bg-blue-100 overflow-hidden md:max-w-full">
+      <div className="md:flex">
+        <div className="p-8 lg:p-16">
+          <h2 className="font-extrabold tracking-wide text-3xl md:text-4xl text-blue-800">
+            {heading}
+          </h2>
+          <p className="mt-2 text-gray-500">{caption}</p>
+          <CalloutBoxButton
+            buttonClickHandler={buttonClickHandler}
+            buttonText={buttonText}
+          />
+        </div>
+        <div className="md:flex-shrink-0 md:h-96">
+          <img
```

Comment:
Currently the image has a space to the right if it's not big enough. Is there a way we could make it so that it would always fill to the edge of the card? 
<img width="1727" alt="Screenshot 2021-07-22 at 15 25 31" src="https://user-images.githubusercontent.com/31065041/126646419-b3fd96f5-8e3c-4d04-ac84-df754eb76f7e.png">

Reply:
Oops :D It was ok on my small screen. 
Fixed, thanks